packages/url/src/index.js

```diff
@@ -47,17 +47,19 @@ export default function url(options = {}) {
             ? path.relative(options.sourceDir, path.dirname(id))
             : path
                 .dirname(id)
-                .split(path.sep)
+                .split(sep)
                 .pop();
 
           // Generate the output file name based on some string
           // replacement parameters
           const outputFileName = fileName
             .replace(/\[hash\]/g, hash)
             .replace(/\[extname\]/g, ext)
-            .replace(/\[dirname\]/g, `${relativeDir}/`)
+            // use `sep` for windows environments
+            .replace(/\[dirname\]/g, `${relativeDir}${sep}`)
             .replace(/\[name\]/g, name);
-          data = `${publicPath}${outputFileName}`;
+          // Windows fix - exports must be in unix format
+          data = `${publicPath}${outputFileName.split(sep).join('/')}`;
```

Comment:
should we pull this `/` into a constant?

Reply:
Could also use `path.posix.sep`

Reply:
`path.posix.sep` is a solid replacement.